### ISSUE TITLE
Feature Request: Improve Accessibiltiy with ARIA tags

### ISSUE THREAD

**Issue by wordshaker** [dotnetnotts/dotnetnotts-web#29]
### Describe the feature

- Add aria tags to the HTML of the website.

### Advantages

- Ensuring the site is more accessible and easier to navigate with screen readers

## Links

- [w3 Org](https://www.w3.org/TR/html-aria/)
- [What the Heck is ARIA? A Beginner's Guide to ARIA for Accessibility](https://www.lullabot.com/articles/what-heck-aria-beginners-guide-aria-accessibility)

#### Acceptance Criteria

Functional and non-functional criteria that should be met by this work

- [ ] Higher accessibility rating
- [ ] Can navigate with screen reader

**Comment by rach-m** [comment]
Hi! Would love to contribute to this!

**Comment by wordshaker** [comment]
Thanks @rach-m! It's all yours 😄
